app/build.gradle

```diff
@@ -331,11 +352,15 @@ dependencies {
     androidTestImplementation 'com.android.support:support-annotations:28.0.0'
     androidTestImplementation 'com.jakewharton.espresso:okhttp3-idling-resource:1.0.0'
     androidTestImplementation 'com.squareup.rx.idler:rx2-idler:0.11.0'
-    testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.0'
+    testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.1'
     androidTestImplementation("androidx.compose.ui:ui-test-junit4:1.0.4")
     debugImplementation("androidx.compose.ui:ui-test-manifest:1.0.4")
     //endregion
 
     //Serialization library
     implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:${libraries.kotlinxserialization}"
-}
\ No newline at end of file
+}
+
+kapt {
+    correctErrorTypes true
+}
```

Comment:
do we need this? what is this for?

Reply:
This was about to make hilt work beside dagger, we were having metadata conflicts. I think for now we can remove, because hilt is out.